app/src/main/res/values/colors.xml

```diff
@@ -13,6 +13,7 @@
     <color name="back_circle_color">#EDEDED</color>
     <color name="indicator_color">#d32f2f</color>
     <color name="progress_secondary_color">#EEEEEE</color>
+    <color name="purple_btn">#6002ee</color>
```

Comment:
@Avjeet can purple cross match with red? Besides on a black background it's hard to see a purple letters and what if we use the same red as in our theme?